src/caselawclient/Client.py

```diff
@@ -706,18 +705,15 @@ def get_properties_for_search_results(self, judgment_uris: list[str]) -> str:
         response = self._send_to_eval(vars, "get_properties_for_search_results.xqy")
         return decode_multipart(response)
 
-    def search_and_parse_results(
-        self, search_parameters: SearchParameters
-    ) -> SearchResults:
+    def search_and_decode_response(self, search_parameters: SearchParameters) -> str:
         response = self.advanced_search(search_parameters)
-        response_string = decode_multipart(response)
-        return SearchResults(response_string)
+        return decode_multipart(response)
 
-    def search_judgments_and_parse_results(
```

Comment:
Wary that the old names might be used somewhere, but I can't find any references to `search_and_parse_results` or `search_judgments_and_parse_results` outside the custom-api-client, so this looks OK.

Reply:
I introduced them in previous PR, the client hasnt beeen released yet with these new functions